modules/openxr/register_types.cpp

```diff
@@ -45,9 +45,13 @@ void preregister_openxr_types() {
 	// For now we create our openxr device here. If we merge it with openxr_interface we'll create that here soon.
 
 	OpenXRAPI::setup_global_defs();
-	openxr_api = OpenXRAPI::get_singleton();
-	if (openxr_api) {
+	if (OpenXRAPI::openxr_is_enabled()) {
+		openxr_api = memnew(OpenXRAPI);
+		ERR_FAIL_NULL(openxr_api);
+
 		if (!openxr_api->initialise(Main::get_rendering_driver_name())) {
```

Comment:
As a side note, this should be renamed to `initialize` with a Z :)

Reply:
Fixed that in a number of places :)